### PR TITLE
Added missing parts

### PR DESCRIPTION
**Pull request by dakyskye**
Then: in `OnPlayerClickMap` callback, it was not checked if a player was in vehicle, thus only player was teleported
Now: it's checked if a player is in the driver seat. If so, then the vehicle gets teleported also.

### CODE REVIEW

**Comment by dakyskye** [comment]
Also, I removed all trailing whitespaces.

File: pawn/stunts-compat/stunts.pwn [hunk 1]
```diff
@@ -309,13 +309,21 @@ hook OnPlayerClickMap(playerid, Float:fX, Float:fY, Float:fZ)
...
 		#pragma unused dialogID, listitem, inputtext
-		
+
 		if (!response)
 		{
 			return Y_HOOKS_CONTINUE_RETURN_0;
 		}
 
-		SetPlayerPos(playerID, fX, fY, fZ);
+		if (GetPlayerVehicleSeat(playerid) == 0)
```
Comment (chromaerror) [inline pawn/stunts-compat/stunts.pwn]: Why is this check using `playerid` when the rest of the checks are using `playerID` with the capitalization?

Comment (dakyskye) [reply pawn/stunts-compat/stunts.pwn]: It should be using `playerID`, you're right. I just somehow missed to modify it. It'd work anyway but still.

File: pawn/stunts-compat/stunts.pwn [hunk 2]
```diff
@@ -309,13 +309,21 @@ hook OnPlayerClickMap(playerid, Float:fX, Float:fY, Float:fZ)
 	inline _response(playerID, dialogID, response, listitem, string:inputtext[])
 	{
 		#pragma unused dialogID, listitem, inputtext
-		
+
 		if (!response)
 		{
 			return Y_HOOKS_CONTINUE_RETURN_0;
```
Comment (Y-Less) [inline pawn/stunts-compat/stunts.pwn]: Generally now the recommended way to do y_hooks returns is `0` or `1` for continue, and `~0` and `~1` for break.  It's much simpler to remember and write.

Comment (dakyskye) [reply pawn/stunts-compat/stunts.pwn]: That's how it was written before. I think modifying all `Y_HOOKS_FOO_RETURN_BAR` in this script deserves another PR.

**Pull request merged**